doc/classes/SceneTree.xml

```diff
@@ -36,22 +36,24 @@
 				[b]Note:[/b] Group call flags are used to control the method calling behavior. By default, methods will be called immediately in a way similar to [method call_group]. However, if the [constant GROUP_CALL_DEFERRED] flag is present in the [param flags] argument, methods will be called with a one-frame delay in a way similar to [method Object.set_deferred].
 			</description>
 		</method>
-		<method name="change_scene">
+		<method name="change_scene_to_file">
 			<return type="int" enum="Error" />
 			<param index="0" name="path" type="String" />
 			<description>
 				Changes the running scene to the one at the given [param path], after loading it into a [PackedScene] and creating a new instance.
 				Returns [constant OK] on success, [constant ERR_CANT_OPEN] if the [param path] cannot be loaded into a [PackedScene], or [constant ERR_CANT_CREATE] if that scene cannot be instantiated.
-				[b]Note:[/b] The scene change is deferred, which means that the new scene node is added on the next idle frame. You won't be able to access it immediately after the [method change_scene] call.
+				[b]Note:[/b] The scene change is deferred, which means that the new scene node is added on the next idle frame. You won't be able to access it immediately after the [method change_scene_to_file] call.
+				[b]Note:[/b] This method used to be named [code]change_scene[/code].
```

Comment:
Add this is unnecessary. Same below.

Reply:
I agree that it’s unnecessary, but I still think it’s a good idea to mention it in the API docs. There’s going to be plenty of tutorials, Godot Engine Q&A posts and other third-party resources that won’t get updated for this change.

Reply:
It only makes sense if we do it consistently for all the renames, and we don't. So no point adding it here.

Once you update the conversion rules, it won't matter much either, as the methods would be correctly updated.

Reply:
> It only makes sense if we do it consistently for all the renames, and we don't. So no point adding it here.

That’s a big exaggeration. If there’s a point to adding a note for all renames, then there would be a point to adding a note for some of the renames. It would just be a smaller point. That being said, consistency is a good thing and I’m not sure whether the good that would come from these notes would outweigh the bad that would come from inconsistency, so I removed them.